commands/bootstrap/__tests__/bootstrap-command.test.js

```diff
@@ -166,6 +167,16 @@ describe("BootstrapCommand", () => {
     });
   });
 
+  describe("with --npm-ci-mode", async () => {
+    const testDir = await initFixture("basic");
+
+    await lernaBootstrap(testDir)("--npm-ci-mode");
```

Comment:
You'll need to pass explicit `"--ci"` here, as the unit test command runner does not decorate the `ci` config like `core/cli` does.

It would also be nice to have a negative test against Yarn, as well as insufficient npm versions. Overwriting `process.env.npm_config_user_agent` with the values from the [example tests](https://runkit.com/embed/auv7hpnsx7f4) would be necessary.